Readme.md

```diff
@@ -31,18 +31,25 @@ gem 'ruby_parser', :require => false, :group => :development
 Add the first language using:
 
 ```Bash
-rake gettext:add_language[XX]
+rake gettext:add_language[xx]
 ```
 
 or
 
 ```Bash
-LANGUAGE=[XX] rake gettext:add_language
+LANGUAGE=[xx] rake gettext:add_language
 ```
 
-where XX is the [ISO 639-1](http://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) 2-letter code for the language you want to create.
+where `xx` is the lowercased [ISO 639-1](http://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) 2-letter code for the language you want to create.
 
-This will also create the `locale` directory (where the translations are being stored) and run `gettext:find` to find any strings marked for translation.
+for example:
+
+```Bash
+rake gettext:add_language[es]
+```
+
+
+This will also create the `locales` directory (where the translations are being stored) and run `gettext:find` to find any strings marked for translation.
```

Comment:
... so this might be a new thing gettext changed ... used to be locale ... will it create locales or config/locales ?

Reply:
With simpl `local` or `locales` rails fails on startup with this message:
```
ruby/3.3.0/lib/ruby/gems/3.3.0/gems/fast_gettext-3.1.0/lib/fast_gettext/translation_repository/base.rb:44:in `find_files_in_locale_folders': path locales could not be found! (RuntimeError)
```

With config/locales, it starts up normally and creates the locales under config/locales:

```ruby
# frozen_string_literal: true

FastGettext.add_text_domain 'app', :path => "config/locales", :type => :po
FastGettext.default_available_locales = ['en','de']
FastGettext.default_text_domain = 'app'
```

```
config/locales/app.pot
config/locales/de
config/locales/de/app.edit.po
config/locales/de/app.po
config/locales/de/app.po.time_stamp
```